pkgs/tools/security/ccid/default.nix

```
@@ -1,34 +1,73 @@
-{ lib, stdenv, fetchurl, pcsclite, pkg-config, libusb1, perl }:
+{ lib
+, stdenv
+, fetchurl
+, flex
+, pcsclite
+, pkg-config
+, libusb1
+, perl
+, zlib
+, gitUpdater
+}:
 
 stdenv.mkDerivation rec {
   pname = "ccid";
-  version = "1.5.2";
+  version = "1.5.4";
 
   src = fetchurl {
     url = "https://ccid.apdu.fr/files/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-E5NEh+b4tI9pmhbTZ8x6GvejyodN5yGsbpYzvrhuchk=";
+    hash = "sha256-boMq3Bcuzc/e4rVvMxRGhIgsvpctr/GTjnqcc6ZPiL8=";
   };
 
   postPatch = ''
     patchShebangs .
     substituteInPlace src/Makefile.in --replace /bin/echo echo
   '';
 
-  preConfigure = ''
-    configureFlagsArray+=("--enable-usbdropdir=$out/pcsc/drivers")
-  '';
+  configureFlags = [
+    "--enable-twinserial"
+    "--enable-serialconfdir=${placeholder "out"}/etc/reader.conf.d"
+    "--enable-ccidtwindir=${placeholder "out"}/lib/pcsc/drivers/serial"
+    "--enable-usbdropdir=${placeholder "out"}/lib/pcsc/drivers"
```

```suggestion
    "--enable-ccidtwindir=${placeholder "out"}/pcsc/drivers/serial"
    "--enable-usbdropdir=${placeholder "out"}/pcsc/drivers"
```

Required by the pcscd NixOS module to build the plugins env